src/components/Dashboard/TaskTable.js

```diff
@@ -0,0 +1,127 @@
+import React, { useMemo, useState } from 'react'
+import {
+  ContextMenu,
+  DataView,
+  DateRangePicker,
+  GU,
+  Link,
+  Text,
```

Comment:
We should avoid using `Text` now that it is deprecated in aragonUI (replaced by `textStyle()`).